README.md

```diff
@@ -102,9 +102,9 @@ id="sanctuary-types-return">[1](#sanctuary-types)</sup>
     - In `a ~> a -> a` (see squiggly arrow above), `a` can be of any type.
       `Semigroup a => a ~> a -> a` adds a constraint such that the type `a`
       must now satisfy the `Semigroup` typeclass. To satisfy a typeclass means
-      to implement any function/method defined by that typeclass. In
-      addition, any implementation of such function/method must also satisfy
-      any laws defined by a given typeclass for the function/method in
+      to implement all functions/methods defined by that typeclass. In
+      addition, any implementation of such functions/methods must also satisfy
+      any laws defined by a given typeclass for the functions/methods in
       question.
```

Comment:
Could we get by with one sentence rather than two?

_To satisfy a type class means to lawfully implement all functions/methods specified by that type class._